### ISSUE TITLE
Site Setup List (checklist): Add checklist step for updating Menus

### ISSUE THREAD

**Issue by gwwar** [Automattic/wp-calypso#40345]
Until we have full site editing available, we should help guide customers on how to add and edit a menu item on their site.

Since doing so currently will take folks to the customizer, let's see if we can pair up with Jeff Rozic on another get started video for adding/editing menus.

<img width="750" alt="image" src="https://user-images.githubusercontent.com/448298/78591949-3ad4f200-7812-11ea-9afd-07287189e43c.png">

### Notes
* The checklist is backed by a server API. We need to know when a task is done (probably the server detecting an edit made to menus). So this will require server patches. It may be also be worth it to make this step **skippable**. (There should be existing examples of this).
* We may need to think about for whom this task is eligible for. (Double check that we don't unset the "doneness" of site setup for folks that have already run through this list).
* Please link to https://wordpress.com/customize/menus/<site> (Same as Edit Menus in quick links)
* Please add required analytics that we expect to this new task

Additional context: pbAPfg-aF-p2

**Comment by amamujee** [comment]
These should also be easily toggled on or off because with Gutenboarding and full site editing they may be handled in signup or in the editor in the future.

**Comment by jancavan** [comment]
@danhauk It's great that we are able to provide video for this step.

Would it be possible to just include a link to the video within the step so it's more optional? My worry about asking them to "watch it first" is it's not obvious that they can actually edit it after, at least from just seeing "Watch video" button, it looks more "required" than optional. Some people may want to jump into editing immediately without have to watch the video.

**Comment by jancavan** [comment]
> I decided against it because it requires extending the functionality of the checklist to have another action. If we're going to do that, I'd rather just make an update like (2) in the original comment.

Between the first option and the second that has two links, the latter feels better, but this makes sense.

The round video thumbnail also feels more in line with our brand. @danhauk

**Comment by lcollette** [comment]
Thanks for exploring what we can do now and where we could go, @danhauk! 

@jancavan covered most of the feedback I had. Including the video is a good move! The circle seems too large for the space allowed and pulls attention away from the primary action. You could try adjusting the size and alignment to help the balance. 

![image](https://user-images.githubusercontent.com/181733/78405330-1fb97680-75b5-11ea-9039-5907c0d7ab57.png)

Let's circle back to this once the flow map is done but just want to say I like this because it makes a single task the focus yet also gives the user context and sets expectations. As we've heard in research sessions, users appreciate having that anchor.  

Does that make sense?
